Add Freebox Home alarm panel (#29487)
home-assistant/home-assistant.github.io: source/_integrations/freebox.markdown

This platform offers you sensors to monitor:
- motion sensor
- door opener 
- plastic cover

## Alarm control panel

> This platform offers you view and control Freebox alarm control panel

Reviewer: ```suggestion
This integration allows you to view and control the Freebox alarm control panel.
```
